README.md

```diff
@@ -16,20 +16,26 @@ You can also run the server and the backend separately; you'll find more info ab
 
 # Dependencies
 
+[Go installed](https://golang.org/doc/install#install), and the [`$GOPATH` properly configured](https://github.com/golang/go/wiki/SettingGOPATH); you can check if it's properly installed and configured obtaining a valid output for these commands:
+```bash
+go version; # prints your go version
+echo $GOPATH; # prints your $GOPATH path
+```
+
```

Comment:
Also, it is better to mention go packages we use:
```
go get github.com/gorilla/handlers
go get github.com/gorilla/mux
go get gopkg.in/bblfsh/client-go.v2
go get gopkg.in/bblfsh/sdk.v1/uast
```

Reply:
Do you think so? It is not usually done in Go (at least in "our projects") because it's handled by go when you do: `go get -u ./...` -> I'll add it to the docs -> do you agree?
Otherwise the dependencies should be maintained by hand, what is a pain ;)

Reply:
@zurk added to the `make build`; it should be automatically done when building the server